app/controllers/katello/api/v2/capsule_content_controller.rb

```diff
@@ -49,9 +49,12 @@ def remove_lifecycle_environment
     api :POST, '/capsules/:id/content/sync', N_('Synchronize the content to the smart proxy')
     param :id, Integer, :desc => N_('Id of the smart proxy'), :required => true
     param :environment_id, Integer, :desc => N_('Id of the environment to limit the synchronization on')
+    param :skip_metadata_check, :bool, :desc => N_('Skip metadata check on each repository on the smart proxy')
     def sync
       find_environment if params[:environment_id]
-      task = async_task(::Actions::Katello::CapsuleContent::Sync, capsule_content, :environment => @environment)
+      task = async_task(::Actions::Katello::CapsuleContent::Sync,
+                        capsule_content,
+                        :environment => @environment, :skip_metadata_check => params[:capsule_content][:skip_metadata_check])
```

Comment:
@jlsherrill should I be using [:capsule_content] with these params? if I do params[:skip_metadata_check] the boolean is returned as a string. What is best practices here?
```
From: /home/vagrant/katello/app/controllers/katello/api/v2/capsule_content_controller.rb @ line 56 Katello::Api::V2::CapsuleContentController#sync:

    53: def sync
    54:   find_environment if params[:environment_id]
    55:   binding.pry
 => 56:   task = async_task(::Actions::Katello::CapsuleContent::Sync,
    57:                     capsule_content,
    58:                     :environment => @environment, :skip_metadata_check => params[:capsule_content][:skip_metadata_check])
    59:   respond_for_async :resource => task
    60: end

[1] pry(#<Katello::Api::V2::CapsuleContentController>)> params
=> {"id"=>"2-almond-proxy-example-com",
 "skip_metadata_check"=>true,
 "api_version"=>"v2",
 "controller"=>"katello/api/v2/capsule_content",
 "action"=>"sync",
 "capsule_content"=>{"id"=>"2-almond-proxy-example-com", "skip_metadata_check"=>true}}

```